src/search-injection/dom.js

```diff
@@ -36,8 +36,15 @@ export const handleRender = ({ docs, totalCount }) => {
             constants.POSITION_KEY,
             'side',
         )
-        const containerID = constants.SEARCH_ENGINES.google.container[position]
-        const container = document.getElementById(containerID)
+        
+        const searchEngine = utils.matchURL
```

Comment:
- You need to call matchURL with the URL as the argument.
- It returns the search engine's name ( which is the key ). Should probably make it return `SEARCH_ENGINE[key]` which is the search engine property object.